tests/foreman/ui/test_syncplan.py

```diff
@@ -83,14 +39,29 @@ def test_positive_create_basic(self, test_data):
 
         """
         with Session(self.browser) as session:
-            make_syncplan(
-                session,
-                org=self.organization.name,
-                name=test_data['name'],
-                description=test_data['desc'],
-                sync_interval=test_data['interval'],
-            )
-            self.assertIsNotNone(self.syncplan.search(test_data['name']))
+            for value in generate_strings_list():
```

Comment:
I would prefer to use something like:
```
for interval in valid_sync_intervals():
    for name in generate_strings_list()
```
And use also different name and description:
```
name=name, description=gen_string('utf8')
```
or something like this